### ISSUE TITLE
double-pass mode switched off = single-pass mode switched on?

### ISSUE THREAD

**Issue by meman102** [vdemichev/DiaNN#386]
Hi, 

When i accidentally put variable modifications to Ac(N-term) and put on double-pass-mode, it says that it can’t run double-pass-mode so it is switched off. Does that mean, that single-pass-mode is automatically switched on? What does it mean for my calculations, if the Neural network classifier is turned off completely?

Thanks in advance!

**Comment by vdemichev** [comment]
Yes, if double pass is switched off, it just means single pass is used. In general, double pass is not recommended anywhere in the docs, so I suggest not to enable it, unless confident that it's beneficial for a particular experiment.

**Comment by Aaron-Amunix** [comment]
Hi Vadim,  loving the software.  In the GUI, it says double-pass gives better data but takes twice as long, is that incorrect?  I have a high end computer running DIA-NN so I can afford the longer analysis time and have been keeping it switched on.  

Appreciate it!
